Solutions/Windows Security Events/Analytic Rules/LocalDeviceJoinInfoAndTransportKeyRegKeysAccess.yaml

```diff
@@ -94,5 +94,5 @@ entityMappings:
     fieldMappings:
```

Comment:
We should split account entity into Name and UPN Suffix to ensure we have a strong identifier to match on

Reply:
Done